src/main/java/core/basesyntax/db/FruitStorage.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Fruit;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FruitStorage {
+    public static final List<Fruit> fruits = new ArrayList<>();
+
+    // Check if the storage already contains such type of fruits
+    // without mentioning their quantity, cause it doesn't matter
+    // That's why boolean equals() doesn't compare quantity for fruits
+    // and why usual boolean contains() doesn't pass in this case
```

Comment:
contains() uses equals() under the hood
so if your equals() doesn't take into account quantity, then it should work